velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -140,24 +158,20 @@ void toDuckDbFilter(
       }
       break;
     }
+    case common::FilterKind::kBigintValuesUsingBitmask: {
+      auto valuesFilter =
+          static_cast<common::BigintValuesUsingBitmask*>(filter);
+      const auto values = valuesFilter->values();
```

Comment:
I guess this is not very performance critical, otherwise we should avoid allocating and copy here.

Reply:
@Yuhta won't the compiler apply RVO here?

Reply:
RVO will save one copy here, you could avoid creating the vector altogether and do something like this:

```c++
template <typename F>
void forEach(F&& f) {
  for (auto i = min_; i <= max_; ++i) {
    if (bitmask_[i - min_]) {
      f(i);
    }
  }
}
```

```c++
valuesFilter->forEach([&](int64_t value) {
  duckFilter->child_filters.push_back(
      constantEqualFilter(makeValue(type, value)));
}
```

But yeah it's obviously overkill if the number of elements in the filter is small.